docs/getting_started/hopsworksai/guides/eks_ecr_integration.rst

```diff
@@ -0,0 +1,319 @@
+
+https://us-east-2.console.aws.amazon.com/ec2/v2/home?region=us-east-2#ModifyInboundSecurityGroupRules:securityGroupId=YOUR_EKS_SECURITY_GROUP_ID
+
+.. _sg-edit-inbound-rules.png: ../../../_images/eks-sg-edit-inbound-rules.png
+.. figure:: ../../../imgs/hopsworksai/eks-sg-edit-inbound-rules.png
+    :alt: Edit inbound rules
+    :target: `sg-edit-inbound-rules.png`_
+    :align: center
+    :figclass: align-center
+
+Add two rules for Http and Https as follows:
+
+.. _sg-hopsworks-inbound-rules.png: ../../../_images/eks-sg-hopsworks-inbound-rules.png
+.. figure:: ../../../imgs/hopsworksai/eks-sg-hopsworks-inbound-rules.png
+    :alt: Edit inbound rules
+    :target: `sg-hopsworks-inbound-rules.png`_
+    :align: center
+    :figclass: align-center
+
+Click Save rules to save the updated rules to the security group.
+
+Step 5: Create a hopsworks cluster with EKS and ECR support
+-----------------------------------------------------------
+
+In Hopsworks.ai, select *Create cluster*:
+
+.. _eks-hopsworks-create-cluster.png: ../../../_images/eks-hopsworks-create-cluster.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster.png`_
+    :align: center
+    :figclass: align-center
+
+Choose the region of your EKS cluster, then click Next:
+
+.. _eks-hopsworks-create-cluster-1.png: ../../../_images/eks-hopsworks-create-cluster-1.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster-1.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster-1.png`_
+    :align: center
+    :figclass: align-center
+
+Choose Enabled to enable the use Amazon EKS and ECR and add your EKS cluster name:
+
+.. _eks-hopsworks-create-cluster-2.png: ../../../_images/eks-hopsworks-create-cluster-2.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster-2.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster-2.png`_
+    :align: center
+    :figclass: align-center
+
+Click Next, choose your SSH Key, click Next and choose the instance profile that you have created in Step 2:
+
+.. _eks-hopsworks-create-cluster-3.png: ../../../_images/eks-hopsworks-create-cluster-3.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster-3.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster-3.png`_
+    :align: center
+    :figclass: align-center
+
+
+Click Next, choose the VPC of your EKS cluster:
+
+.. _eks-hopsworks-create-cluster-4.png: ../../../_images/eks-hopsworks-create-cluster-4.png
+.. figure:: ../../../imgs/hopsworksai/eks-hopsworks-create-cluster-4.png
+    :alt: Create Hopsworks cluster
+    :target: `eks-hopsworks-create-cluster-4.png`_
+    :align: center
+    :figclass: align-center
+
+Click Next, choose one of the subnets in the VPC:
```

Comment:
I would say to select the first or add something to reassure people that it does not matter which one they choose.